src/main/notifications/index.test.js

```diff
@@ -16,9 +16,10 @@ import {localizeMessage} from 'main/i18nManager';
 
```

Comment:
It might be worth adding some additional tests to check for the behaviour of the notification changes. One example could be making sure it calls the right stuff per OS.